lib/dartino.dart

```diff
@@ -10,10 +10,9 @@ import 'package:atom/atom.dart';
 import 'package:atom/node/process.dart';
 import 'package:atom/utils/disposable.dart';
 import 'package:atom_dartino/sdk/sdk.dart';
+import 'package:atom_dartino/usb.dart';
```

Comment:
Done as part of a later commit https://github.com/dartino/atom-dartino/commit/723e50a2a1e900e3a7e41fa5d9e29a6faa955e70